scality_manila_utils/helper.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) 2015 Scality
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import io
+import os
+import os.path
+import signal
+
+from scality_manila_utils import utils
+from scality_manila_utils.export import ExportTable
+from scality_manila_utils.exceptions import (EnvironmentException,
+                                             ExportException)
+
+
+class Helper(object):
+    """
+    Handles addition, removal of exports as well as client permissions.
+    """
+
+    def __init__(self, root_volume, exports_path):
+        """
+        Create a new Helper instance operating on the given exports file.
+
+        :param root_volume: root volume exported by the sfused nfs connector
+        :type root_volume: string (unicode)
+        :param exports_path: path to nfs exports file
+        :type exports_path: string (unicode)
+        """
+        # Ring root volume export
+        self.root_volume = root_volume
+
+        if not os.path.exists(exports_path):
+            raise EnvironmentException('Unable to locate exports file')
+
+        # Path to nfs exports file
+        self.exports_path = exports_path
+
+        with io.open(self.exports_path, 'rt') as exports_file:
+            lines = (line for line in exports_file)
```

Comment:
So, `lines` = `exports_file` from a behaviour PoV? Then I guess `exports_file` can be passed to `ExportTable.deserialize` as-is :-)